core/drivers/imx/dcp/dcp_utils.c

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2020 NXP
+ */
+#include <dcp_utils.h>
+#include <drivers/imx/dcp.h>
+#include <kernel/cache_helpers.h>
+#include <malloc.h>
+#include <mm/core_memprot.h>
+#include <trace.h>
+
+struct __packed mem_hdr {
```

Comment:
Why is this `__packed`?

Reply:
I guess it's a copy paste from the CAAM driver. It's useless here